p/themes/base-theme/template.rtl.css

```diff
@@ -59,6 +59,10 @@ p {
 	font-size: 1em;
 }
 
+p.help {
+	margin: 5px 0 0.5em !important;
```

Comment:
Could you try to find a rule without `!important` ?

Reply:
Same.